credoai/evidence/containers.py

```diff
@@ -108,3 +113,22 @@ def to_evidence(self, **metadata):
     def _validate(self, df):
         if list(df.columns) != ["results"]:
             raise ValidationError("Profiler data must only have one column: 'results'")
+
+
+class ModelProfilerContainer(EvidenceContainer):
+    """Container for Model Profiler type evidence"""
+
+    def __init__(self, df, labels=None, metadata=None):
```

Comment:
Is forcing everything to a df above (in `evaluate()`) a good idea? 

Why not just a basic dictionary all along, since that's the basic data type wrapped in ModelProfilerEvidence objects?

Reply:
`EvidenceContainer` the container's class abstract enforces a `DataFrame` input. 

The idea of containers is to provide a unifying layer to collect the output of any evaluator, while at the same time keeping evidence internal structure separated. It is slightly repetitive, but you get the freedom of manipulating evidence structure as you please. 

DataFrames have a more defined structure than dictionaries (which helps standardization) plus when you reconvert to dictionary, you can do so in multiple ways according to what best fit the evidence.